### DISCUSSION TITLE
Summer 2022 Survey Report (2207RL)

### DISCUSSION THREAD

**Discussion by kstierhoff** [SWFSC/estimATM#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jrenfree** [comment]
Here is the current version of the 2022 Summer Survey report. I'm still waiting to obtain the rest of the processed acoustic data before updating many of the results summaries, but wanted to get input on some of the changes that I made compared to the 2021 report.
[reportSurvey_2207RL.pdf](https://github.com/kstierhoff/estimATM/files/9923610/reportSurvey_2207RL.pdf)

- I replaced Figure 4 with a habitat map from a single timepoint at the beginning of the survey. Then, I relocated the figure of the accomplished transects overlayed on the time-aggregated habitat map to the Results section (now Figure 11). How do you feel about this change?
- For Figure 12, is it necessary to include the transect lines and/or the vessel navigation? I don't feel that it adds up, and if anything makes the figure caption uneccessarily complicated when referencing solid/dashed grey and black lines.
- For the calibration results, I trimmed down a lot of what parameters are presented. Currently, the calibration results are taken from the template .ecs files that were used to process the acoustic data. Moreover, for the Lasker results which show the beam pattern, I still use the single-target data from the .xml files. Those, however, are computed using differing calibration parameters. Do you think this is worth fixing? If so we will need to discuss how best to format and read the calibration data.
- In Appendix B, should we include a summary of the Shimada CTD casts, since those were used for processing the Lisa Marie and Saildrone data?

**Comment by kstierhoff** [comment]
A few thoughts:

   - I think the revised Fig 4 is OK, given those were the conditions
   closest to when the survey plan was finalized, but I don't know if it
   really has much bearing on the survey design. That is, we define the survey
   area based on historic habitat distribution, so I'm not sure this figure
   adds a lot and I'm not even sure that the sentence beginning on line 92 is
   even accurate (i.e., the transects don't extend into Canada, where a lot of
   potential habitat existed) . I really like the aggregated map to indicate
   the habitat conditions at that time that we actually collected the data,
   and I believe that map should be in the Results as you have it. If we keep
   Fig 4 as-is, I think we need to explain why the survey area didn't extend
   north to Vancouver Is.
   - I think that having either the vessel tracks or sampled acoustic
   transects provides context to the station sampling. I think you could
   remove the vessel path (gray line) if it simplified things.
   - I like the reduced calibration tables. If they're adequate for someone
   to use our .raw files and get the same results, then I'd say we're good.
   I'm not sure how much is involved in "fixing" the single target detection
   data. Honestly, I'm not sure how critical it is to have the beam pattern
   mapped. That could be one of those things where we included the figures
   because I made them, not because they were useful/informative. Your
   thoughts on that?
   - I think we should add an appendix for SH casts, since they're used for
   processing data.


On Wed, Nov 2, 2022 at 12:43 PM jrenfree ***@***.***> wrote:

> Here is the current version of the 2022 Summer Survey report. I'm still
> waiting to obtain the rest of the processed acoustic data before updating
> many of the results summaries, but wanted to get input on some of the
> changes that I made compared to the 2021 report.
> reportSurvey_2207RL.pdf
> <https://github.com/kstierhoff/estimATM/files/9923610/reportSurvey_2207RL.pdf>
>
>    - I replaced Figure 4 with a habitat map from a single timepoint at
>    the beginning of the survey. Then, I relocated the figure of the
>    accomplished transects overlayed on the time-aggregated habitat map to the
>    Results section (now Figure 11). How do you feel about this change?
>    - For Figure 12, is it necessary to include the transect lines and/or
>    the vessel navigation? I don't feel that it adds up, and if anything makes
>    the figure caption uneccessarily complicated when referencing solid/dashed
>    grey and black lines.
>    - For the calibration results, I trimmed down a lot of what parameters
>    are presented. Currently, the calibration results are taken from the
>    template .ecs files that were used to process the acoustic data. Moreover,
>    for the Lasker results which show the beam pattern, I still use the
>    single-target data from the .xml files. Those, however, are computed using
>    differing calibration parameters. Do you think this is worth fixing? If so
>    we will need to discuss how best to format and read the calibration data.
>    - In Appendix B, should we include a summary of the Shimada CTD casts,
>    since those were used for processing the Lisa Marie and Saildrone data?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kstierhoff/estimATM/discussions/8#discussioncomment-4041287>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARG62TD4YXHHHEGN2YRRE3WGK75VANCNFSM6AAAAAARVHLJIE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Kevin L. Stierhoff

Research Fisheries Biologist, NOAA/NMFS/SWFSC

**Reply by jrenfree** [reply]
Thanks, and I think I agree with most of these ideas.

> I think the revised Fig 4 is OK, given those were the conditions
   closest to when the survey plan was finalized, but I don't know if it
   really has much bearing on the survey design. That is, we define the survey
   area based on historic habitat distribution, so I'm not sure this figure
   adds a lot and I'm not even sure that the sentence beginning on line 92 is
   even accurate (i.e., the transects don't extend into Canada, where a lot of
   potential habitat existed) . I really like the aggregated map to indicate
   the habitat conditions at that time that we actually collected the data,
   and I believe that map should be in the Results as you have it. If we keep
   Fig 4 as-is, I think we need to explain why the survey area didn't extend
   north to Vancouver Is.

Yeah if we aren't actively choosing transect lines based on the pre-survey habitat, then probably easiest to just remove this figure and any reference to it. Agreed that the figure in the Results section is nice and should remain.

> I think that having either the vessel tracks or sampled acoustic
   transects provides context to the station sampling. I think you could
   remove the vessel path (gray line) if it simplified things.

Will try it without the vessel path.

> I like the reduced calibration tables. If they're adequate for someone
   to use our .raw files and get the same results, then I'd say we're good.
   I'm not sure how much is involved in "fixing" the single target detection
   data. Honestly, I'm not sure how critical it is to have the beam pattern
   mapped. That could be one of those things where we included the figures
   because I made them, not because they were useful/informative. Your
   thoughts on that?

The beam pattern maps are more of a niceity than a necessity, and they certainly do look cool. They do give a sense of the quality of the calibration, so not completely useless. Another option is to export the single-target detections in Echoview using the correct calibration parameters, then the survey report would simply need to read that CSV file and plot the compensated target strength, which is already calculated for us. I can submit that as a feature request and we can discuss there. 

> I think we should add an appendix for SH casts, since they're used for
   processing data.

Easy enough. Thanks.
